exercises/e-data02-derive-master.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.            c_diff=gr.IQR(DF["diff"]/DF["diff_median"])
</pre><p>The IQR stuff is overkill! You can remove this (I was being nuts before). Let's just do mean, sd, and coefficient of variation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdelrosario/evc-course/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>